doc/getting-started/building-the-node-on-windows.md

```diff
@@ -1,104 +1,159 @@
+See [install.md](./install.md) for a more thorough explanation of the dependencies and packages that have to be installed to run a node. Although the installation instructions in that document are aimed at building on Linux, large parts of the text are interesting to Windows users as well.
 
-You can run all the instructions that invoke `choco` in PowerShell with root privileges. We recommend installing and using `git-bash` for development purposes, which is `git` when installed with `choco install git`.
+## Install the Haskell environment
 
-## Install GHC
+The recommended way to install the Haskell tools is via [GHCup](https://www.haskell.org/ghcup/). Check [this page](https://www.haskell.org/ghcup/install/) for further explanation on the installation process.
 
-The recommended way is to run the following command in PowerShell:
+Once GHCup is installed, open a new terminal (to get an updated environment) and run:
 
-```PowerShell
-choco install --version 8.10.7 ghc
+```bash
+ghcup install ghc 8.10.7
+ghcup install cabal 3.6.2.0
+ghcup set ghc 8.10.7
+ghcup set cabal 3.6.2.0
+```
+
+Alternatively, with `ghcup tui` you can pick the specific versions of the tools that you want to install, in particular you should have installed and set:
+- `cabal >= 3.6.2.0`
+- `GHC >= 8.10.7`
+
+## Install MSYS2 and basic dependencies
+​
+### GHCUP and MSYS2
+​
+Note that the GHCUP comes with its own installation for MSYS2, and it can be used to build `cardano-node`. You can start the MSYS2 shell with
+```cmd
+<ghcup-path>\msys64\msys2_shell.cmd -defterm -here -no-start -mingw64
+```
+​
+Make sure to enable `mingw64` subsystem. It is also possible to enable `mingw64` subsystem by
+​
+```bash
+source shell mingw64
+```
+​
+​
+The MSYS2 shell is bundled with `pacman` package manager.
+​
+```bash
+pacman -Syu
+pacman -S --needed base-devel automake git
+export PATH=$PATH:/mingw64/bin:<ghcup-path>/bin:<ghcup-path>
+```
+​
+You can update the `mingw64` toolchain, that add number of packages including latest `gcc`.
+​
+```bash
+pacman -S mingw-w64-x86_64-toolchain
+```
+​
+### Installing MSYS independently
+​
+Go to [the MSYS2 website](https://www.msys2.org/) and follow its instructions to install MSYS2. It will also install `pacman`. Then, update and upgrade all `pacman` packages and install basic dependencies:
+​
+```
+pacman -Syu
+pacman -S --needed base-devel mingw-w64-x86_64-toolchain git
 ```
 
-## Install pkg-config
+## Install Libsodium and LMDB
 
-```PowerShell
-choco install pkgconfiglite
+```
+pacman -S mingw-w64-x86_64-libsodium
+pacman -S mingw-w64-x86_64-lmdb
 ```
 
-## Install vcpkg
+## Install Secp256k1
 
-You will first need to install `vcpkg` to proceed with `libsodium` library installation (which is the next step). 
+```
+pacman -S unzip
```

Comment:
Didn't we want to install it via `vcpkg`? @yogeshsajanikar

Reply:
The latest version includes alternative instructions for `vcpkg`